src/core/utils/index.js

```diff
@@ -328,6 +328,25 @@ export const findFileForPlatform = ({ userAgentInfo, platformFiles }) => {
   return (platform && platformFiles[platform]) || platformFiles[OS_ALL];
 };
 
+export const getlastUpdated = ({
```

Comment:
We already have a function named `getVersionInfo()` that does the same thing: https://github.com/mozilla/addons-frontend/search?utf8=%E2%9C%93&q=getVersionInfo&type=. 

I'd use it instead of creating a new function. It is already used in `AddonMoreInfo`, so only `AddonHead` would need a change.